### PR TITLE
Milestone2: components

### PR DESCRIPTION
**Pull request by blakbox23**
In this part of the project I : 
- Set up a class component `Calculator`,
- Implemented the required calculator design,
- Imported the Calculator component in the `App.js` component.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,9 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Math magicians
-        </a>
-      </header>
-    </div>
+    <Calculator />
   );
 }
```
Comment (Emmyn5600) [inline src/App.js]: - [x] Great job you have imported the Calculator.js in the App.js file, however, The App.js component should be class-based.
storage1

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+
```
Comment (Emmyn5600) [inline src/components/Calculator.js]: - [x] Kindly, fix eslint errors without disabling line of code.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @blakbox23 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
-  Git/GitHub flow best practices have been followed.:heavy_check_mark:
- Great design matching with the given design:heavy_check_mark:

Changes required :writing_hand: 
  
   - Please check the comment under the review.

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/components/Calculator.js (hunk 2 above)
Comment (blakbox23) [reply src/components/Calculator.js]: 👍🏾 done.

File: src/App.js (hunk 1 above)
Comment (blakbox23) [reply src/App.js]: @Emmyn5600 Thank you, I got it.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @blakbox23,

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**